Zend/tests/closure_067.phpt

```diff
@@ -14,7 +14,17 @@ class Bar {
 
 $c = (new Bar())->baz();
 $r = new \ReflectionFunction($c);
+// Closures are not inside of a namespace, thus the short name is the full name.
 var_dump($r->getShortName());
+// The namespace is empty.
+var_dump($r->getNamespaceName());
+// The function is not inside of a namespace.
+var_dump($r->inNamespace());
+// And the namespace name + the short name together must be the full name.
+var_dump($r->getNamespaceName() . ($r->inNamespace() ? '\\' : '') . $r->getShortName() === $r->getName());
```

Comment:
```suggestion
// And the short name must be the full name.
var_dump($r->getShortName() === $r->getName());
```

as the namespace is tested above already

Reply:
No. I very intentionally want to check the namespace + short = full name contract as an end-to-end test here.

Reply:
I understand, but doing so means you do not trust PHP.